microsetta_interface/templates/source.jinja2

```diff
@@ -123,20 +123,22 @@
         <div class="row mt-4">
         {% for detail in remote_surveys %}
             <div class="col-12 col-sm-6 col-md-4 p-2">
-            {% if not detail.answered %}
+            {% if detail.survey_template_id == 10005 or not detail.answered %}
```

Comment:
Can we replace instances of 10005 with the appropriate constant? It helps with code readability/comprehension?